amqp_broker.go

```diff
@@ -71,17 +71,23 @@ func NewAMQPConnection(host string) (*amqp.Connection, *amqp.Channel) {
 }
 
 // NewAMQPCeleryBroker creates new AMQPCeleryBroker
-func NewAMQPCeleryBroker(host string) *AMQPCeleryBroker {
-	return NewAMQPCeleryBrokerByConnAndChannel(NewAMQPConnection(host))
+func NewAMQPCeleryBroker(host string, queue ...string) *AMQPCeleryBroker {
```

Comment:
The code is structured in certain way in order to support backward compatibility and this breaks backward compatibility. There is major version branch v2 in works with completely new api so take a look there.

Reply:
nitpick: it should be `queues` not `queue.

Reply:
Variadic arguments are optional, so this can still be called without the queue parameter, as you can see in the tests the previous tests are still working, and we can specify a queue if we want to, that is why it is called `queue`, as you can see in the code, I only use the first one.

Reply:
You are right. I skimmed through and I missed that.
It still feels a bit messy. Do we foresee any other optional parameters added in the future?

Reply:
Not that I can think of, in addition variadic have to be the last parameter, which does get things pretty much *stuck* like this. One way to avoid it would be to have add  like #112 as well as del (in case we want a single queue and not the default plus others), and for it to be implemented for redis as well as amqp. That would also mean we will want to BLPOP multiple queues. You would prefer new helpers than optional parameters?